### ISSUE TITLE
[FEATURE] Increase Short Rewards

### ISSUE THREAD

**Issue by KeinNiemand** [Mirror-Protocol/mirror-contracts#83]
**Problem definition**
Premiums are pretty high right now so we should increase short rewards

**Feature specification**
Increase the maximum short reward ratio to somthing higher then 40%

**Comment by csanti** [comment]
Before modifying this parameter I think we would need a governance proposal.

**Comment by KeinNiemand** [comment]
Well I saw a few people on the forums and on discord talking about increating this and they said that creating a governance proposal would be useless since actaully changing it would require a change in the code, that's why I created this issue and the PR. 
The actual solution would be to make this directly changable trough governance.

**Comment by csanti** [comment]
Anyways, we would need all MIR holders to vote on this. Otherwise, this will be changeable through governance

**Comment by hmouhtar** [comment]
@csanti 
So if a text proposal to increate short rewards passes, the team will implement that change?
Do you think it'd be a positive change?

**Comment by csanti** [comment]
yes, if a text proposal passes it would be easier to add the change. I think it's better to discuss the change and its impact on the forums.
